README.md

```diff
@@ -22,11 +22,12 @@ See the section about [running tests](https://facebook.github.io/create-react-ap
 
 ## How to build docker image
 <i>Install Docker on your machine</i>
+Pull nginx image: <code>docker pull nginx</code>
```

Comment:
Do we need to pull this explicitly? When you specify `FROM nginx` you already specify which image we are basing on and it should be pulled automatically when you try to build the image of your container.